css/style.css

```diff
@@ -0,0 +1,103 @@
+* {
```

Comment:
- [ ] Please use inline comments to give your review. This project requires you to use GitHub features. This is an example of inline comments. Kindly check[ here](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md#how-to-use-github-features) on how to give inline comments.

- [Optional] I would suggest that you give all your comments as separate bullet points so they can be easily identified from one another.🙃